package.json

```diff
@@ -13,6 +13,7 @@
         "sandbox:ext": "pnpm -dir examples/sandbox run dev:ext",
         "build": "turbo run build --filter=./packages/*",
         "test": "turbo run test --filter=./packages/*",
+        "test:e2e": "PW_CHROMIUM_ATTACH_TO_OTHER=1 playwright test -c playwright.config.ts",
```

Comment:
This won't work on windows
Better assign it in the config with `process.env['PW_CHROMIUM_ATTACH_TO_OTHER'] = '1'` or use `cross-env`